ui/src/app/machines/views/MachineDetails/MachineStorage/ChangeStorageLayout/ChangeStorageLayout.tsx

```diff
@@ -69,8 +68,7 @@ export const ChangeStorageLayout = ({ systemId }: Props): JSX.Element => {
     </div>
   ) : (
     <FormCard data-test="confirmation-form" sidebar={false}>
-      <FormikForm
-        buttons={FormCardButtons}
+      <FormikForm<Record<string, never>>
```

Comment:
Seems a bit odd but I suppose it's correct. I guess it would be nice if that was the default or something.

Reply:
Yeah it's a shame to have to explicitly define the form values type this way. I was thinking of adding a couple of type aliases to at least make it easier to understand like:
```
export type AnyObject = Record<string, unknown>;
export type EmptyObject = Record<string, never>;
```